_posts/2022-08-24-kubeflow-1.6-release.md

```diff
@@ -0,0 +1,143 @@
+toc: false
+comments: true
+image: images/logo.png
+hide: false
+categories: [release]
+permalink: /kubeflow-1.6-release/
+author: "Kubeflow 1.6 Release Team"
+---
+
+The Kubeflow Community is excited to announce the availability of the Kubeflow v1.6 software release, which includes
+ support for Kubernetes v1.22 and introduces an alpha release of the Kubeflow Pipeline v2 (KFP v2) functionality. 
+
+Kubeflow v1.6 also adds new hyperparameter support for the population based algorithm in Katib, and provides a combined
+ Python SDK for PyTorch, MXNet, MPI, XGBoost in Kubeflow’s distributed Training Operator. For model serving, v1.6 has
+ new ClusterServingRuntime and ServingRuntime CRDs, and a new Model Spec was introduced to the InferenceService
+ Predictor Spec, which provides a new way to specify models in KServe 0.8. Additionally, 1.6 cleans up a few CVEs in
+ the central dashboard and enables PodDefaults webhook to pick-up new updates to certificates.   For the Kubernetes upgrade,
+ the community developed and tested each Kubeflow component manifest for Kubernetes v1.22.  As v1.22 introduced some breaking changes, the upgrade was a team effort, and this Kubeflow release management process will be useful in the community's future updates of Kubernetes and other software dependencies.  
+ 
+In parallel to developing these new software features, the Kubeflow Community also completed its annual user survey.
+ The survey generated many good user insights into requirements for Kubeflow, and you can see more on the
+ [survey results](https://docs.google.com/document/d/1TLl8iFWQ-x2pVAHV9G8GFYMgAOuN2VFOnZV2JQNA-uc/edit#). Of the many
+ highlights, we would like to identify the growing request for model monitoring as shown in the chart below.
+
+![2022 User Survey Gaps In ML Activities and Workflow](/images/2022-08-24-kubeflow-1.6-release/2022-user-survey-gaps-in-ml-activities-and-workflow.png)
+
+We believe the increased focus on monitoring identifies Kubeflow’s maturing user base. The Community is making efforts
+ to explain Kubeflow’s current functionality and build into the users’ new model monitoring requirements. In addition,
+ the user survey results and Kubeflow Working Group roadmaps will be discussed in the upcoming Kubeflow Summit. You can
+ learn more about the Kubeflow Summit [here](https://docs.google.com/document/d/1fLg1EqVtJekcXxT8GM_Rqg0-G-vIDVCrzuxE06Oq9dI/edit).
+
+To round back to the current software delivery, the following takes a deeper look at the v1.6 highlights in more detail.
+
+## Kubeflow v1.6 details
+
+From a new feature introduction standpoint, Kubeflow v1.6 includes an experimental release of the KFP v2’s new front-end,
+ back-end and SDK, which deliver a modern UI and DAG, first class support for metadata, and a simplified component
+ authoring experience. This slide deck, [KFP v2 Introduction](https://docs.google.com/presentation/d/1HzMwtI2QN67xQp2lSxmuXhitEsukLB7mvZx4KAPub3A/edit#slide=id.gecbd775238_0_20),
```

Comment:
changed name of slide deck to match the actual name of the presentation (KFP v2 Introduction)